### PR TITLE
SEO-171719-Links-With-Redirects-MAUI

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: MAUI/Introduction/Overview.md [hunk 1]
```diff
@@ -26,6 +26,6 @@ Similar to the Knowledge Base, the Forum section also contains responses to ques
 
 ## Support and feedback
 
-If you are unable to find the information that you are looking for in the self-help resources mentioned above, please contact us by creating a [support ticket](https://www.syncfusion.com/support/directtrac/incidents).
+If you are unable to find the information that you are looking for in the self-help resources mentioned above, please contact us by creating a [support ticket](https://www.syncfusion.com/Account/Login?ReturnUrl=%2fsupport%2fdirecttrac%2fincidents).
```
Comment (MallikaRK) [inline MAUI/Introduction/Overview.md:29]: https://support.syncfusion.com/create - This is our latest URL. Please make a note of it. Change the url.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: MAUI/Introduction/Overview.md (hunk 1 above)
Comment (berylouma) [reply MAUI/Introduction/Overview.md]: Updated the correct URL

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**